### PR TITLE
test

### PR DESCRIPTION
**Pull request by axil42**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+| type = Women
+| country = Mali
+| coach = Amara Traoré
+| fiba_ranking = 25
+| team colours =  
+| joined_fiba = 1961
+| fiba_zone = FIBA Africa
+| national_fed =  Fédération Malienne de Basketball
+| oly_appearances = 1 ([[Basketball at the 2008 Summer Olympics – Women's tournament|2008]])
+| oly_medals =
```
Comment (axilleas) [inline README.md]: comment

Comment (axil42) [inline README.md]: ok

File: README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+===FIBA Africa Championship for Women 2007===
+Mali traveled to Senegal for the [[FIBA Africa Championship for Women 2007]] qualifying tournament for the [[Basketball at the 2008 Summer Olympics|2008 Summer Olympics]] in Beijing. The squad won the first round with a 4–1 record, losing only to host Senegal. Mali also had the highest positive point differential of any squad in the tournament. In the ensuing rounds, Mali beat [[Cameroon women's national basketball team|Cameroon]], followed by [[Angola women's national basketball team|Angola]]. In the championship match, Mali beat Senegal 63–56, qualifying automatically for the 2008 Olympics for the first time. Captain [[Hamchétou Maïga]] was voted the MVP of the tournament, while teammate [[Diéné Diawara]] grabbed the most rebounds.
+
+==Results==
```
Comment (axilleas) [inline README.md]: asdf

File: README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+* [[2005 FIBA Africa Championship for Women|2005]] : 5th
+* [[2007 FIBA Africa Championship for Women|2007]] : 1st
+* [[2009 FIBA Africa Championship for Women|2009]] : 2nd
+* [[2011 FIBA Africa Championship for Women|2011]] : 3rd
+* [[2013 FIBA Africa Championship for Women|2013]] : 5th
+* [[2015 FIBA Africa Championship for Women|2015]] : 5th
+
+==Olympic roster==
+{{FIBA roster header|team=Mali Women's National Basketball Team
+| color1 = Red| bg1 = Green| color2 = Gold| bg2 = Green }}
```
Comment (axilleas) [inline README.md]: **ddd**

File: README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+==References==
+{{reflist}}
+
+==External links==
+*8http://www.femabasket.com/ Official website]
+*[http://www.fiba.com/pages/eng/fc/FIBA/fibaStru/nfLeag/p/langlc/en/nationalfederationnumber/324/nfProf.html FIBA profile]
+
+{{FIBA Africa women's teams}}
+
```
Comment (axilleas) [inline README.md]: > what?

File: README.md (hunk 2 above)
Comment (axil42) [inline README.md]: ok

File: README.md (hunk 3 above)
Comment (axil42) [inline README.md]: ok

File: README.md (hunk 4 above)
Comment (axil42) [inline README.md]: > this

**Pull request merged**